docs/user/HTTP_REDIRECT.md

```diff
@@ -0,0 +1,107 @@
+# HTTP Redirects
+
+The [HTTPRoute][] resource can issue redirects to clients or rewrite paths sent upstream using filters. Note that
+HTTPRoute rules cannot use both filter types at once. Currently, Envoy Gateway only supports core [HTTPRoute filters][]
+which consist of `RequestRedirect` and `RequestHeaderModifier`. To learn more about HTTP routing, refer to the
+[Gateway API documentation][].
+
+Follow the steps from the [Secure Gateways](SECURE_GATEWAY.md) to install Envoy Gateway and the example manifest. Do not
+proceed until you can curl the example backend from the Quickstart guide using HTTPS.
+
+## Redirects
+Redirects return HTTP 3XX responses to a client, instructing it to retrieve a different resource. A
+[`RequestRedirect` filter][req_filter] instructs Gateways to emit a redirect response to requests that match the rule.
+For example, to issue a permanent redirect (301) from HTTP to HTTPS, configure `requestRedirect.statusCode=301` and
+`requestRedirect.scheme="https"`:
+```shell
+cat <<EOF | kubectl apply -f -
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: http-to-https-filter-redirect
+spec:
+  parentRefs:
+    - name: eg
+  hostnames:
+    - redirect.example
+  rules:
+    - filters:
+      - type: RequestRedirect
+        requestRedirect:
+          scheme: https
+          statusCode: 301
```

Comment:
As a heads up, if you don't set `statusCode` at all it will default to 301 (because in the xds if you don't configure a specific status code then Envoy will use 301 by default. Gateway API only specifies support for `301` and `302` here so really you only need to include `statusCode: 302`, otherwise you can leave it out.

That much detail isn't necessary for the docs, but might be nice to call out that if you don't configure the `statusCode` then it has a default.

Reply:
@AliceProxy thanks for the review and feedback. We're setting default values in other docs. For example, see the Gateway listener in the Quickstart Guide. I'll leave the example as-is and add a note below it.